pkgs/by-name/to/toml-sort/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+let
+  version = "0.23.1";
+in
+python3Packages.buildPythonPackage {
```

```suggestion
python3Packages.buildPythonApplication {
```
It seems `toml-sort` is mainly used as a cli, in this case we prefer `buildPythonApplication`